src/answers-utilities.ts

```diff
@@ -2,6 +2,15 @@ import { DisplayableFacet } from '@yext/answers-core';
 import { isLevenshteinMatch } from './utils/searchable-facets';
 
 export default {
+  /**
+   * Searches through the specified facet and filters out the options that aren't a
+   * Levenshtein match for the given searchTerm.
```

Comment:
I'm not sure that we need to include the matching algorithm (Levenshtein) is these docs